pkg/apis/kops/componentconfig.go

```diff
@@ -293,6 +293,8 @@ type KubeAPIServerConfig struct {
 	RequestheaderAllowedNames []string `json:"requestheaderAllowedNames,omitempty" flag:"requestheader-allowed-names"`
 	// FeatureGates is set of key=value pairs that describe feature gates for alpha/experimental features.
 	FeatureGates map[string]string `json:"featureGates,omitempty" flag:"feature-gates"`
+	// MaxRequestsInflight The maximum number of non-mutating requests in flight at a given time.
+	MaxRequestsInflight int `json:"maxRequestsInflight,omitempty" flag:"max-requests-inflight"`
```

Comment:
So you need to add `flag-empty:"0"` here (although separately we might want to figure out if 0 should be omitted I guess!)

Also technically this should be int32 (or int64), because it's part of our API and we shouldn't have architecture dependent types in our API.   Not that it matters in practice :-)